index.html

```diff
@@ -0,0 +1,231 @@
+                <li>2021</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tonic-courses">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="btn">
+              <button class="btn-tonic" type="submit">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-myself-card">
+        <div class="all-components">
+          <!-- <div class="first-image">
+            <img src="./asset/Snapshoot Portfolio.png" alt="" />
+          </div> -->
+          <div class="tonic-content">
+            <h1>
+              About <br />
+              Myself
+            </h1>
+
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h3 class="about-myself-connect">LET'S CONNECT</h3>
+            <div class="tonic-courses">
+              <ul>
+                <li>
+                  <a href="#"><i class="fab fa-twitter"></i></a>
+                </li>
+                <li>
+                  <a href="#"><i class="fab fa-facebook-f"></i></a>
+                </li>
+                <li>
+                  <a href="#"><i class="fab fa-github"></i></a>
+                </li>
+                <li>
+                  <a href="#"><i class="fas fa-hand-peace"></i></a>
+                </li>
+                <li>
+                  <a href="#"><i class="fab fa-gitlab"></i></a>
+                </li>
+              </ul>
+            </div>
+            <div class="btn">
+              <button class="btn-tonic" type="submit">Get my resume</button>
+            </div>
+            <div class="languages">
+              <p>Languages</p>
+              <a><i class="fas fa-chevron-down"></i></a>
+            </div>
+            <div class="abt-myself-courses">
+              <a href=""><img src="./asset/fram9.png" alt="" /></a>
+              <a href=""><img src="./asset/frame10.png" alt="" /></a>
+              <a href=""><img src="./asset/frame11.png" alt="" /></a>
+            </div>
+            <div class="languages">
+              <p>Frameworks</p>
+              <a><i class="fas fa-chevron-right"></i></a>
+            </div>
+            <hr />
+            <div class="languages">
+              <p>Skills</p>
+              <a>
+                <i class="fas fa-chevron-right"> </i>
+              </a>
+            </div>
+            <hr />
+          </div>
```

Comment:
- [ ] Kindly note that the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li` to follow the project requirement😉
- [ ] Kindly make sure the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`🎉